From 66ec51afb8329bce7b0bb87785a886fb803e8f29 Mon Sep 17 00:00:00 2001
From: Wander Boessenkool <wboessen@redhat.com>
Date: Fri, 7 Aug 2009 11:05:02 +0200
Subject: [PATCH 05/33] add suspend quirk for a Acer Aspire One 751

Added suspend quirk for a Acer Aspire One 751 (with weird poulsbo
graphics when using the psb driver for Xorg).
---
 .../10freedesktop/20-video-quirk-pm-acer.fdi       |    7 +++++++
 1 files changed, 7 insertions(+), 0 deletions(-)

diff --git a/fdi/information/10freedesktop/20-video-quirk-pm-acer.fdi b/fdi/information/10freedesktop/20-video-quirk-pm-acer.fdi
index 91ad85c..561b413 100644
--- a/fdi/information/10freedesktop/20-video-quirk-pm-acer.fdi
+++ b/fdi/information/10freedesktop/20-video-quirk-pm-acer.fdi
@@ -161,6 +161,13 @@
         </match>
       </match>
 
+      <!-- Aspire One 751 -->
+      <match key="system.hardware.product" string="AO751h">
+        <merge key="power_management.quirk.dpms_on" type="bool">true</merge>
+        <merge key="power_management.quirk.vbemode_restore" type="bool">true</merge>
+        <merge key="power_management.quirk.vbestate_restore" type="bool">true</merge>
+      </match>
+
       <!-- Aspire One 110 -->
       <match key="system.hardware.product" prefix_outof="AOA110;AOA150">
         <merge key="power_management.quirk.none" type="bool">true</merge>
-- 
1.7.1

